tensorflow/lite/tools/evaluation/stages/inference_profiler_stage.h

```diff
@@ -32,13 +32,15 @@ namespace evaluation {
 
 // An EvaluationStage to profile a custom TFLite inference config by comparing
 // performance in two settings:
-// 1. User-defined TfliteInferenceParams (The 'test' setting)
-// 2. Default TfliteInferenceParams (The 'reference' setting)
-// The latter essentially implies single-threaded CPU execution.
+// 1. The 'test' setting: User-defined TfliteInferenceParams
+// 2. The 'reference' setting: User-defined or default TfliteInferenceParams
+// The default essentially implies single-threaded CPU execution.
 class InferenceProfilerStage : public EvaluationStage {
  public:
   explicit InferenceProfilerStage(const EvaluationStageConfig& config)
       : EvaluationStage(config) {}
```

Comment:
When this constructor is called, the tool fails when trying to initialize the `reference_stage_`.

The error is:
```
E0618 02:09:57.517596    3384 tflite_inference_stage.cc:109] TfliteInferenceParams not provided
tensorflow/lite/tools/evaluation/stages/inference_profiler_stage_test.cc:67: Failure
```

Maybe you need to initialize the reference config as it is done [here][1]?

[1]: https://github.com/tensorflow/tensorflow/pull/62937/files#diff-8e9daef4424254eca5ebca48cddcb99b8c2b2ae16a80dd817d396edfe1b02a57L83-L91